### ISSUE TITLE
Add support for search and replace in include components

### ISSUE THREAD

**Issue by NathanLovato** [GDQuest/product-packager#84]
This is about adding the possibility to search and replace text in included code, for example, to hide the fact that we use different class names in course material (e.g. turn `Item_v2` into `Item`) on the website.

The search and replace itself is already implemented in https://github.com/GDQuest/product-packager/blob/master/mdx-utils/src/md/preprocessor.nim, but what's missing is parsing JSX expressions when parsing MDX components.

So, this depends on #65

**Comment by NathanLovato** [comment]
Superseded by #91
